packages/perseus/src/util/parse-perseus-json/exhaustive-test-tool/index.ts

```diff
@@ -97,11 +97,17 @@ async function testFile(path: string, outputDir: string) {
 }
 
 function getAssessmentItemData(raw: unknown): unknown {
```

Comment:
🤔 Typically I haven't seen articles referred to as Assessment Items. Just thinking through this, I might be too picky since this is a testing tool, but I wonder if we'd be better off exposing a new parsing function specifically for parsing articles (which are just an array of `parsePerseusRenderer` results).  We will need that for integration in consuming apps anyways (ie. article rendering wouldn't use `parsePerseusItem()` but rather `parsePerseusArticle()` ideally).

This might also help the error output make more sense as I was a little jarred when seeing `.question.` in the error output.. articles don't have questions. :)